tests/gcm/util/test_plotting.py

```diff
@@ -0,0 +1,23 @@
+import networkx as nx
+import numpy as np
+import pandas as pd
+
+from dowhy.gcm.util import plot, plot_adjacency_matrix
+
+
+def test_when_plot_does_not_raise_exception():
+    plot(nx.DiGraph(edges=[('X', 'Y'), ('Y', 'Z')]))
+
+
+def test_plot_adjacency_matrix():
+    causal_graph = pd.DataFrame(np.array([[0, 1, 0], [0, 0, 1], [0, 0, 0]]),
+                                columns=['X', 'Y', 'Z'],
+                                index=['X', 'Y', 'Z'])
+
+    # Check if calling the method causes some import or runtime errors
+    plot_adjacency_matrix(causal_graph, is_directed=True)
+    # TODO: Plotting undirected graphs with networkx causes an error when an older networkx version is used with a newer
```

Comment:
do we use undirected graphs anywhere?

Reply:
if this functionality is required, another option to bump up the networkx requirement to be 2.4+

Reply:
We have undirected graphs as part of our causal discovery algorithms (e.g. identify the skeletons). This method helped to visualize them. However, currently in DoWhy, we might only have directed edges. Should we assume that for now and remove the option to plot undirected graphs?

Reply:
That sounds good to me

Reply:
Was checking the code again, I think it should be fine to simply keep the option to plot undirected graphs as well. A user would not need to specify this, it will be inferred from the type of graph (e.g. ```DiGraph``` vs ```Graph```). Overall, the user only needs to call ```plot(causal_model)```, e.g. ```plot(nx.DiGraph([('X', 'Y')])) # plots X -> Y```